homeassistant/components/sma/entities.py

```diff
@@ -0,0 +1,675 @@
+"""Entity Descriptions for the sma integration."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import (
+    STATE_CLASS_MEASUREMENT,
+    STATE_CLASS_TOTAL_INCREASING,
+    SensorEntityDescription,
+)
```

Comment:
Please use `SensorStateClass` and `SensorDeviceClass` enums
```suggestion
from homeassistant.components.sensor import (
    SensorDeviceClass,
    SensorEntityDescription,
    SensorStateClass,
)
```